x-pack/plugins/lists/server/services/items/find_list_item.ts

```diff
@@ -27,6 +28,13 @@ import {
   transformElasticToListItem,
 } from '../utils';
 
+export const getTotalHitsValue = (totalHits: number | { value: number } | undefined): number =>
```

Comment:
You can use `rest_total_hits_as_int` in the search endpoint to say if you want it to be returned an integer.

Reply:
It doesn't seem to fix the type error which is the main reason for doing this. We do a similar transform elsewhere in the code with equivalent requests